.github/workflows/publish.yml

```diff
@@ -0,0 +1,51 @@
+name: publish
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'Version to Checkout and Release'
+        required: true
+        default: 'v0.1.2'
+
+jobs:
+  publish:
+    strategy:
+      matrix:
+        go-version: [1.13.x]
+        os: [ubuntu-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+      -
+        uses: actions/checkout@master
+        with:
+          repository: rakyll/hey
+          ref: ${{ github.event.inputs.tag }}
```

Comment:
If you need a tag, we could always have a text file in the repo that can be read from disk. This may be necessary if a build fails and we can't keep in sync with upstream.

Reply:
You said you didn't understand this comment on Slack.

How about reading the tag from a file in the repo?

Reply:
Yes I'm just thinking where we would get the tag into the file. Is that going to be something that you add in a commit to this repo, and it just does a checkout with that version? If so why can't we do a `git tag vX.X.X` and a `git push origin vX.X.X`? If the build fails then we have the "Rerun jobs" button. I'm not just sure if this is what you mean?